cre.cpp

```diff
@@ -436,7 +436,7 @@ static int setCallback(lua_State *L) {
 
 static int initCache(lua_State *L) {
     const char *cache_path = luaL_checkstring(L, 1);
-    int cache_size = luaL_checkint(L, 2);
+    unsigned long long int cache_size = luaL_checklong(L, 2);
```

Comment:
Overkill, it's actually a macro that just does `(long) luaL_checkinteger`, and `lua_Integer`is `ptrdiff_t`, so this is never going to be 64 bits wide on 32 bits arches ;).

TL;DR: `long` ;).

Reply:
replaced with lvsize_t